### PR TITLE
Formatting and code organization; event console updates without reloa…

### PR DESCRIPTION
**Pull request by ryan-s-lee**
…d on events.

### CODE REVIEW

**Comment by ryan-s-lee** [comment]
Actually wait a sec Edit Points stopped working

**Review by ryanyychen — APPROVED** [review]

File: frontend/src/Components/Events/CustomizableEventConsole.svelte [hunk 1]
```diff
@@ -167,23 +84,49 @@
...
+            indexedRows.forEach((row) => {
+                row["Edit Points"] = {
+                    onclick: async () => {
+                        modalUserData = row;
+                    },
+                    text: "Edit Points",
+                    args: [],
+                };
+            });
+            console.log(indexedRows);
```
Comment (ryanyychen) [inline frontend/src/Components/Events/CustomizableEventConsole.svelte:108]: Can you remove this console.log line?

File: frontend/src/Components/Events/CustomizableEventConsole.svelte [hunk 2]
```diff
@@ -193,107 +136,148 @@
...
-                                            disabled="true"
-                                            style="background-color: gray;"
-                                            >
-                                            {object[property].text}
-                                        </button>
-                                    {:else}
-                                        <button
-                                            on:click={object[property].onclick.apply(
+                                            );
+                                            console.log("this gets printed");
```
Comment (ryanyychen) [inline frontend/src/Components/Events/CustomizableEventConsole.svelte:242]: remove this log statement

**Review by ryanyychen — CHANGES_REQUESTED** [review]

File: frontend/src/Components/Events/CustomizableEventConsole.svelte (hunk 2 above)
Comment (ryan-s-lee) [reply frontend/src/Components/Events/CustomizableEventConsole.svelte]: Done

File: frontend/src/Components/Events/CustomizableEventConsole.svelte (hunk 1 above)
Comment (ryan-s-lee) [reply frontend/src/Components/Events/CustomizableEventConsole.svelte]: Also done

**Review by ryanyychen — APPROVED** [review]

**Pull request merged**